Misc/NEWS.d/next/Core and Builtins/2022-11-25-09-34-57.gh-issue-99761.sRqH37.rst

```diff
@@ -0,0 +1 @@
+Add optimized method ``_Py_is_valid_index`` to calculate valid indices
```

Comment:
Please remove the NEWS entry. This change doesn't affect Python users. I will add "skip news" label.